exercicios/para-casa/resolucao/classes/Person.js

```diff
@@ -0,0 +1,9 @@
+export class Person {
```

Comment:
Vi que vc acabou importando para a CLient. Teve algum motivo pra criar essa classe Person? 



Sugestao de insercao tudo na class CLient:

`export class Client{
    
        name
        #cpf
        banks
    
        constructor(name, cpf){
            this.name = name
            this.#cpf = cpf 
            this.banks = []
        }
`